08-reactivity/02-scroll-after-update/components/MiniMessenger.vue

```diff
@@ -44,7 +46,21 @@ export default {
       this.newMessage = '';
     },
   },
-};
+
+  watch: {
+    async ['messages.length']() {
+      const list = this.$refs.items[0]?.parentElement;
+
+      if (!list) {
+        return;
+      }
+
+      await nextTick();
+
+      list.scrollTop = list.scrollHeight;
+    },
+  },
+});
```

Comment:
Better, but it is still not true. If a user removes some middle message, does it mean, that they want to scroll to the bottom? If a new message came, does it mean, that the user want to scroll to the bottom. 

Here the source of scrolling to the bottom is a certain action - sending a new message.